library-client-rtsp/src/main/java/com/alexvas/rtsp/codec/VideoDecodeThread.kt

```diff
@@ -33,52 +35,46 @@ class VideoDecodeThread (
             val widthAlignment = capabilities.widthAlignment
             val heightAlignment = capabilities.heightAlignment
             Pair(
-                Util.ceilDivide(width, widthAlignment) * widthAlignment,
-                Util.ceilDivide(height, heightAlignment) * heightAlignment)
+                    Util.ceilDivide(width, widthAlignment) * widthAlignment,
+                    Util.ceilDivide(height, heightAlignment) * heightAlignment)
         }
     }
 
     override fun run() {
         if (DEBUG) Log.d(TAG, "$name started")
 
-        val decoder = MediaCodec.createDecoderByType(mimeType)
-        val widthHeight = getDecoderSafeWidthHeight(decoder)
-        val format = MediaFormat.createVideoFormat(mimeType, widthHeight.first, widthHeight.second)
+        try {
+            val decoder = MediaCodec.createDecoderByType(mimeType)
+            val widthHeight = getDecoderSafeWidthHeight(decoder)
+            val format = MediaFormat.createVideoFormat(mimeType, widthHeight.first, widthHeight.second)
 
-        decoder.setOnFrameRenderedListener(onFrameRenderedListener, null)
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
Library min SDK is Android M (ext.min_sdk_version = 24). What is the reason for that line?

Reply:
I needed this in my project but it's not necessary here